distributed/distributed.yaml

```diff
@@ -275,13 +275,15 @@ distributed:
       nframes: 0
       ignore-modules:
         - asyncio
+        - datashader
         - functools
         - dask
         - distributed
         - coiled
         - cudf
         - cuml
         - prefect
+        - pytest
```

Comment:
pytest is apparently a little more involved https://github.com/dask/distributed/pull/8198

```suggestion
```